ansible.cfg

```diff
@@ -1,2 +1,6 @@
 [ssh_connection]
-pipelining = True
\ No newline at end of file
+pipelining = True
+[defaults]
+stdout_callback = json
+callback_whitelist = json
+json_indent = 4
```

Comment:
These changes do not matter, as the cfg file is not used. Look at the description to use it in the future